client/signup/reskinned-processing-screen/index.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@automattic/react-i18n';
+import { useInterval } from 'lib/interval/use-interval';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+// Total time to perform "loading"
+const DURATION_IN_MS = 6000;
+
+/**
+ * This component is cloned from the CreateSite component of Gutenboarding flow
+ * to work with the onboarding signup flow.
+ */
+export default function ReskinnedProcessingScreen( { hasPaidDomain } ) {
+	const { __ } = useI18n();
+
+	const steps = React.useRef(
```

Comment:
I'm curious to know why we have used `useRef` over a simple array. :)

Reply:
@aneeshd16  Good question. The variable created from `useRef` works like a static variable that lives in the entire lifecycle of the component. So the component dones't need to call the translate function everytime it renders. For more details please see the following link.
https://reactjs.org/docs/hooks-reference.html#useref

> However, useRef() is useful for more than the ref attribute. It’s handy for keeping any mutable value around similar to how you’d use instance fields in classes.

Reply:
Got it, this is a good tip!